### ISSUE TITLE
Type Error: 'setting_kwargs'

### ISSUE THREAD

**Issue by aaravnavani** [nicklashansen/dmcontrol-generalization-benchmark#14]
Hello,

I am facing the same issue as the one described [here](https://github.com/nicklashansen/dmcontrol-generalization-benchmark/issues/10). I ran this command ```python3 src/train.py   --algorithm svea   --seed 0``` but got this error ```TypeError: load() got an unexpected keyword argument 'setting_kwargs'```. 

I did install dm_control which comes with mujoco 2.2.0 and I dont see any way to install mujoco 2.0.0 from the[ mujoco download page](https://github.com/deepmind/mujoco/releases). 

@nicklashansen Any help would be appreciated.

**Comment by nicklashansen** [comment]
Hi, MuJoCo 2.0.0 is still available for download from the legacy site: https://www.roboti.us/download.html
Note that you will need a license for this version, which is provided for free on the same site. Let me know if this does not solve your problem!

**Comment by aaravnavani** [comment]
Hi, thanks for the reply. I downloaded MuJoCo 2.0.0 from the legacy website above with the license. Currently, on my system, I have MuJoCo 2.2.0 (this came when I installed dm control with pip). I tried removing mujoco 2.2.0 using ```pip3 uninstall mujoco``` and then I moved the downloaded mujoco 2.0.0 folder from the legacy website into the directory ```/home/anaconda3/envs/dmcgb/lib/python3.7/site-packages``` (where the original mujoco install was) and when I ran ```
python3 src/train.py   --algorithm svea   --seed 0```, I got this error ```ModuleNotFoundError: No module named 'mujoco'``` Is this the correct way to use the newly downloaded Mujoco 2.0.0 or am I missing something? 

Thanks!
